src/app/modules/registration/pages/danger-obs/danger-obs.page.ts

```diff
@@ -35,14 +35,18 @@ export class DangerObsPage extends BasePage {
     super(RegistrationTid.DangerObs, basePageService, activatedRoute);
   }
 
+  get GeoHazardName(): string {
```

Comment:
Dette er en kopi av løsningen i AddOrEditDangerObsModalPage og bryter med OnceAndOnlyOnce.
Jeg foreslår at du heller lager en felles løsning for dette i KdvService, slik at vi ikke trenger å tenke på dette når vi jobber med jordskjema. 
Fiks gjerne også stor forbokstav i funksjonsnavnet. Det ser ut som C# og er ikke vakkert;)